.env.example

```diff
@@ -0,0 +1,2 @@
+# variable name follow: REACT_APP_<name-of-your-variable>
+REACT_APP_TITLE=React Boilerplate
```

Comment:
Is this really everything you have in your .env? Where is the port configuration, backend configuration, etc?

Reply:
added more in .env.example

Reply:
looks good to me